src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	
+	private String plusSign = "+ ";
+	private String minusSign = "- ";
+	
+	
+	private Person taggedPerson;
+	private Tag tagObject;
+	private boolean shouldAdd;
+	
+	public Tagging(Person person, Tag tag, boolean add) {
+		this.taggedPerson = person;
+		this.tagObject = tag;
+		this.shouldAdd = add;
+	}
+
+	@Override
+	public String toString() {
```

Comment:
Try not to override the existing methods if it is not required or the purpose of the existing method does not match well.
You can read the Official Documentation to get information